mscore/seq.cpp

```diff
@@ -654,12 +666,12 @@ void Seq::addCountInClicks()
 //    methods like semaphores can also not be used.
 //-------------------------------------------------------------------
 
-void Seq::process(unsigned n, float* buffer)
+void Seq::process(unsigned framesPerPeriod, float* buffer)
```

Comment:
FYI, *nothing changed in this function other than improved the names of variables*.  Previous variables names were conflicting and misleading.